cli/mender-artifact/artifacts.go

```diff
@@ -164,7 +164,7 @@ func unpackArtifact(name string) (string, error) {
 	}
 
 	inst := r.GetHandlers()
-	if ! (len(inst) == 1 && inst[0].GetUpdateType() == "rootfs-image" ) {
+	if !(len(inst) == 1 && inst[0].GetUpdateType() == "rootfs-image") {
```

Comment:
I personally think it is better to split style changes from actual code changes. This will ease the git archeology in the future.

Reply:
(this is for future reference, you don't need to do it now)